### ISSUE TITLE
Dark mode text appears as the wrong color

### ISSUE THREAD

**Issue by mortocks** [henninghall/react-native-date-picker#483]
**Describe the bug**

The text for dark model on iOS appears to be the wrong color

**Expected behavior**

When in dark mode, the text color for the picker should be white (same as the default iOS picker)

**To Reproduce**
Add example code that reproduces the behavior. 
```javascript
export default class App extends Component {

  state = { date: new Date() }

  render = () =>
    <DatePicker
      theme='dark' // This setting should make the text white
      date={this.state.date}
      onDateChange={date => this.setState({ date })}
    />

}
```

Note: Yes you can change the color by overriding `textColor` but I would expect the default mode to select good defaults. The term `theme` implies a collection of colors working together rather than just setting the background color.

![CleanShot 2022-03-24 at 10 07 15](https://user-images.githubusercontent.com/314196/159816972-299751ea-62cc-44ec-b897-65a3e9a3ce3b.png)

**Smartphone (please complete the following information):**
 - iOS simulator iPhone 13
 - React Native version [e.g. 0.66.4]
 - react-native-date-picker version [e.g. 4.2.0]

**Comment by Majiedo** [comment]
yeah I have the same thing here, you can change the `textColor="white"` prop like I did but I have one problem with this work around is when I want the user have ability to change to dark mode or light mode but It's hard to track that and change the "textColor" prop!

**Comment by parallaxpro** [comment]
const colorScheme = Appearance.getColorScheme();

textColor={colorScheme === 'dark' ? 'white' : 'black'}

**Comment by henninghall** [comment]
Fixed in v4.2.4
